.github/workflows/prerelease.yml

```diff
@@ -0,0 +1,154 @@
+name: Prerelease pipeline
+
+on:
+  release:
+    types:
+      - prereleased
+    tags:
+      - 'v*'
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  GO_VERSION: '1.9.7'
+  GPG_MAIL: 'infrastructure-eng@newrelic.com'
```

Comment:
since these secrets are only used for nix packaging, I've moved them for that step only. 
Same thing with the PFX_* (only used in windows packaging). 
It limits a bit the exposure of the secrets, but it's nothing critical